tests/checkpointing/test_model_checkpoint.py

```diff
@@ -131,7 +131,7 @@ def configure_optimizers(self):
         assert mc_specific_data['current_score'] == score
 
         lr_scheduler_specific_data = chk['lr_schedulers'][0]
-        assert lr_scheduler_specific_data['_step_count'] == epoch + 2
+        assert lr_scheduler_specific_data['_step_count'] == epoch + 1
```

Comment:
actually it should be +2 since epoch here is zero indexed (0, 1, 2) and default_value for `_step_count` is 1. so for total_epochs=1 it should be 1(default) + (epoch(0) + 1) = 2

Reply:
changed back :]